packages/botonic-cli/package.json

```diff
@@ -40,8 +40,7 @@
     "/lib",
     "/npm-shrinkwrap.json",
     "/oclif.manifest.json",
-    "/scripts",
-    "/templates"
```

Comment:
Why removing `/templates`?

Reply:
these are not included anymore, as we have moved them to `botonic-examples`